options/locale/locale_en-US.ini

```diff
@@ -245,7 +245,7 @@ register_helper_msg = Already have an account? Sign in now!
 social_register_helper_msg = Already have an account? Link it now!
 disable_register_prompt = Registration is disabled. Please contact your site administrator.
 disable_register_mail = Email confirmation for registration is disabled.
-remember_me = Remember Me
+remember_me = Remember this Device
```

Comment:
Should probably also rename the token here.

Reply:
Rename would need to happen across all locales. Not sure how that works with crowdin, so I don't wanna touch that..

Reply:
Reason I'm asking is because I wonder if crowdin will properly invalidate other translations if you just change the english translation like that.

Reply:
It will invalidate correctly, no problem with that

Reply:
Crowdin invalidates the translations once you change the value - you don't need to change the key.